### PR TITLE
V2.0 back button

### PR DESCRIPTION
**Pull request by jansepar**
Fixes back button issue for IE10 and Firefox raised by issue https://github.com/mobify/mobifyjs/issues/137.

Started writing a regression test, but was thwarted by the fact that the `history` object is shared between parent and iframes, thus I could not simulate `history.back()`...

I also noticed when debugging this issue that we broke Mobify post-flood (post document.write). Not good! This was determined to be broken during the merge of the swift feature branch. This has been fixed, and a regression test has been added.

@noahadams could you review plz?

### CODE REVIEW

File: tests/mobify-library.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html class="foo">
+<head>
+  <meta charset="utf-8">
+  <title>capture.js tests</title>
```
Comment (noahadams) [inline tests/mobify-library.html]: o rly?

**Comment by noahadams** [comment]
The use of the symlink for the mobifyjs directory causes the tests not to be runnable natively on windows (though the capture.js, and mobify-library tests pass on IE10 when loaded _from_ an external machine that resolves the symlink properly.

Consequently, running `grunt test` on windows doesn't work either. 

I wouldn't characterize these as blockers however.

**Comment by jansepar** [comment]
@noahadams added a note about the Windows issue it in the README, which should be sufficient for now.

File: tests/mobify-library.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+<script>
+    // http://api.qunitjs.com/QUnit.config/
+    QUnit.config.autostart = false;
+    QUnit.config.testTimeout = 30 * 1000;
+
+    QUnit.start();
+
+
+    /* 
+     * All rendering engines (aside from Webkit) remove all objects
```
Comment (noahadams) [inline tests/mobify-library.html]: Maybe more like "Most browsers create a new global javascript namespace in the new document, except webkit, which preserves the namespace as it was before the call to document.open()"

Comment (jansepar) [inline tests/mobify-library.html]: Sure.

**Comment by noahadams** [comment]
address my comment and your comment and you'll get your cherished +1

**Comment by noahadams** [comment]
:+1:

**Pull request merged**